sdk/notificationhubs/notification-hubs/src/utils/notificationUtils.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Notification } from "../models/notification.js";
+
+/**
+ * @internal
+ */
+export function createMultipartDirectNotification(
+  notification: Notification,
+  deviceHandles: string[]
+): string {
+  return (
+    "--nh-batch-multipart-boundary\r\n" +
```

Comment:
if I was still a SDET I'd make an event with the string `--nh-batch-multipart-boundary` somewhere in it ;)

Actually, out of curiosity if someone did take a user string and a malicious user used the boundary string, could they trick your SDK into sending another event to somewhere else?

Reply:
Fixed to randomize the boundaries